docs/advanced_topics/adding_signals.rst

```diff
@@ -0,0 +1,68 @@
+==================
+How to use Signals
+==================
+
```

Comment:
Can we start with a line that points the user to the Django documentation on signals?
https://docs.djangoproject.com/en/3.1/topics/signals/